src/ethereum/mod.rs

```diff
@@ -20,10 +20,16 @@ use rpassword;
 use std::time::Duration;
 use web3::transports::Http;
 use web3::types::Block as Web3Block;
+use web3::types::H256 as Web3H256;
+use web3::types::U128 as Web3U128;
+use web3::types::U256 as Web3U256;
```

Comment:
We should not need to rename those anymore...

Reply:
Agree, I will remove them except `Web3Block`